docs/concepts.rst

```diff
@@ -752,6 +752,67 @@ Note that these can be used in conjunction with ``depends_on_past`` (boolean)
 that, when set to ``True``, keeps a task from getting triggered if the
 previous schedule for the task hasn't succeeded.
 
+One must be aware of the interaction between trigger rules and skipped tasks
+in schedule level. Skipped tasks will cascade through trigger rules 
+``all_success`` and ``all_failed`` but not ``all_done``, ``one_failed``, ``one_success``,
+``none_failed`` and ``dummy``. 
+
+For example, consider the following DAG:
+
+.. code:: python
+
+  #dags/branch_without_trigger.py
```

Comment:
If we have this file in repo, we should include it instead of copy it's content. It's allow to keep example allways working and fresh. 
See: 
http://www.sphinx-doc.org/en/1.5/markup/code.html#directive-literalinclude
https://github.com/apache/airflow/blob/master/docs/howto/operator.rst

Reply:
This is a very good idea. But there is no such folder for those scripts. What do you think if we make a dir under docs, like /docs/example_dags/branch_without_trigger.py ?

Reply:
@cixuuz I don't think create a py file iin docs is a good idea.

Reply:
Considering this kind of script is only for demo and no other script storing in .py, I agree with u that not doing so. Thank you!

Reply:
Files can be stored in the `airflow/example_dags` directory.
Example: 
```
.. literalinclude:: ../../airflow/example_dags/example_python_operator.py
    :language: python
    :start-after: [START howto_operator_python_kwargs]
    :end-before: [END howto_operator_python_kwargs]
```
Source: https://raw.githubusercontent.com/apache/airflow/master/docs/howto/operator.rst

Reply:
I will try to extract all examples to separate files and create automatic tests.

Reply:
@mik-laj , I don't think this should be a blocker as we have done it for other doc files as well. If we want to extract this file from doc and put it in example folder, we could do it in a later pr.